src/main/java/org/mifos/connector/mpesa/camel/CamelProperties.java

```diff
@@ -0,0 +1,64 @@
+package org.mifos.connector.mpesa.camel;
+
+/**
+ * Central Definition of all the Camel Exchange Properties
+ */
+public class CamelProperties {
```

Comment:
I think file not used, instead src/main/java/org/mifos/connector/mpesa/camel/config/CamelProperties.java is getting used. Merge them into one with the ones you need.

Reply:
noted

Reply:
done